[2024-10-20T03:07:12.952+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: crypto_etl.load manual__2024-10-20T03:07:05.820649+00:00 [queued]>
[2024-10-20T03:07:12.965+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: crypto_etl.load manual__2024-10-20T03:07:05.820649+00:00 [queued]>
[2024-10-20T03:07:12.966+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2024-10-20T03:07:12.985+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): load> on 2024-10-20 03:07:05.820649+00:00
[2024-10-20T03:07:12.996+0000] {standard_task_runner.py:57} INFO - Started process 237 to run task
[2024-10-20T03:07:12.999+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'crypto_etl', 'load', 'manual__2024-10-20T03:07:05.820649+00:00', '--job-id', '229', '--raw', '--subdir', 'DAGS_FOLDER/crypto_etl_dag.py', '--cfg-path', '/tmp/tmp89ohtgo4']
[2024-10-20T03:07:13.000+0000] {standard_task_runner.py:85} INFO - Job 229: Subtask load
[2024-10-20T03:07:13.056+0000] {task_command.py:416} INFO - Running <TaskInstance: crypto_etl.load manual__2024-10-20T03:07:05.820649+00:00 [running]> on host c03d178edf0e
[2024-10-20T03:07:13.144+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='crypto_etl' AIRFLOW_CTX_TASK_ID='load' AIRFLOW_CTX_EXECUTION_DATE='2024-10-20T03:07:05.820649+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-10-20T03:07:05.820649+00:00'
[2024-10-20T03:07:13.205+0000] {logging_mixin.py:154} INFO - Success: Loaded 101 recall records to recalls_db.
[2024-10-20T03:07:13.206+0000] {python.py:194} INFO - Done. Returned value was: None
[2024-10-20T03:07:13.218+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=crypto_etl, task_id=load, execution_date=20241020T030705, start_date=20241020T030712, end_date=20241020T030713
[2024-10-20T03:07:13.252+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-10-20T03:07:13.271+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
